pkg/virt-controller/services/virtiofs.go

```diff
@@ -15,6 +15,32 @@ import (
 	"kubevirt.io/kubevirt/pkg/virtiofs"
 )
 
+// RequireAnyVirtiofsPrivilegedContainer Returns true if any of the returns true if any
```

Comment:
nit: `RequireAnyVirtiofsPrivilegedContainer Returns true if any of the returns true` -> `RequireAnyVirtiofsPrivilegedContainer returns true`

Reply:
was to make sure that people understand that it returns true :-P